### ISSUE TITLE
Pretrained models for CIFAR10

### ISSUE THREAD

**Issue by zhangman164** [clovaai/overhaul-distillation#22]
Dear Author,
I find that both of the two pt files are prepared for CIFAR100, could you please provide a link to a  trained model for CIFAR10?

**Comment by bhheo** [comment]
Hi, thank you for your interest in our work.
I'm sorry but since we did't test our method on CIFAR-10, we don't have any trained model for CIFAR-10.

**Comment by zhangman164** [comment]
Thank you, and what if I would like to train a teacher net from scratch? I have tried to use the WRN-net here to train a teacher net, however, it could not match the train process. Could you  please tell me which work you used to train the teacher net?

**Comment by bhheo** [comment]
The code that I used to train teacher network is almost same as current code.

If you remove distillation loss and change `d_net` to `s_net`, then it becomes training codes for teacher network.
So, you have to change 
https://github.com/clovaai/overhaul-distillation/blob/392a006560c2ab3aca8bb8fc7b1bf826298d9fd6/CIFAR-100/train_with_distillation.py#L88-L91
this lines.

I think previous issue #11 will help you to train the teacher network.
